scripts/common_test.sh

```diff
@@ -0,0 +1,8 @@
+
+[ "$EUID" -ne 0 ] && echo "Run as root" && exit 1
```

Comment:
I like if's better, but, really, up to you.

Reply:
Makes sense, more readable.